src/dispatch/feedback/service/models.py

```diff
@@ -21,14 +20,17 @@ class ServiceFeedback(TimeStampMixin, FeedbackMixin, Base):
     feedback = Column(String)
     rating = Column(String)
     schedule = Column(String)
-    hours = Column(Integer)
+    hours = Column(Numeric(precision=10, scale=2))
     shift_start_at = Column(DateTime)
     shift_end_at = Column(DateTime)
 
     # Relationships
     individual_contact_id = Column(Integer, ForeignKey("individual_contact.id"))
     individual = relationship("IndividualContact")
```

Comment:
I'd move this relationship to the `IndividualContact` model and make it backref, like the one we have for events

`events = relationship("Event", backref="individual")`